android/app/build.gradle

```diff
@@ -91,6 +91,7 @@ dependencies {
     compile 'com.jakewharton:butterknife:8.4.0'
     compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    compile 'net.opacapp:multiline-collapsingtoolbar:1.3.1'
```

Comment:
The method count looks quite high to me.
http://www.methodscount.com/?lib=net.opacapp%3Amultiline-collapsingtoolbar%3A1.3.1

Isn't there an alternative solution though?